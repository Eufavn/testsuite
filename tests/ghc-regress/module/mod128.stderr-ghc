
./Mod128_A.hs:2:16: Warning: `T' is exported by `T' and `T(Con)'
