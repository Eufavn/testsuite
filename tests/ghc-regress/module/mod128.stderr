
Mod128_A.hs:2: Warning: `T' is exported by `T' and `T(Con)'
